### PR TITLE
tools: add a newly tool for creating release announce template for Twitter

### PR DESCRIPTION
**Pull request by komainu8**
This tool output the following template.

```
PGroonga 3.0.1: 2023-04-27 リリース！ https://groonga.org/ja/blog/2023/04/27/pgroonga-3.0.1.html 今回のリリースでは、
PGroonga 3.0.1: 2023-04-27 has been released! https://groonga.org/en/blog/2023/04/27/pgroonga-3.0.1.html In this release,
```

This tool get the version of PGroonga and the release date in GitHub release page.
Also, this tool confirm created URL whether valid or not.

### CODE REVIEW

File: tools/create-release-announce-template-for-twitter.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -ue
+
+PGROONGA_LATEST_VERSION_AND_DATE=$(curl https://api.github.com/repos/pgroonga/pgroonga/releases/latest | jq '.name' | sed -e 's/^"//' -e 's/"$//')
```
Comment (kou) [inline tools/create-release-announce-template-for-twitter.sh:5]: `jq -r '.name'`

File: tools/create-release-announce-template-for-twitter.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+PGROONGA_LATEST_VERSION=$(curl https://api.github.com/repos/pgroonga/pgroonga/releases/latest | jq '.tag_name' | sed -e 's/^"//' -e 's/"$//')
+
+RELEASE_BLOG_BASE_URL="https://groonga.org"
+RELEASE_BLOG_DATE=$(echo $PGROONGA_LATEST_VERSION_AND_DATE | awk -F ":" '{print $2}' | sed -e 's/^ //' -e 's/"$//' | sed -e 's/-/\//g')
```
Comment (kou) [inline tools/create-release-announce-template-for-twitter.sh:9]: Why do you prefer `YYYY/MM/DD` to `YYYY-MM-DD`? `YYYY-MM-DD` is the ISO 8601 format.

Comment (kou) [reply tools/create-release-announce-template-for-twitter.sh:9]: Ah, you want a path.

Comment (kou) [reply tools/create-release-announce-template-for-twitter.sh:9]: `RELEASE_BLOG_DATE_PATH` may be better.

Comment (kou) [reply tools/create-release-announce-template-for-twitter.sh:9]: `echo $PGROONGA_LATEST_VERSION_AND_DATE | cut -d ' ' -f 3 | tr - /`

**Comment by komainu8** [comment]
@kou Thank you for your comments. I've fixed them.

File: tools/create-release-announce-template-for-twitter.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  exit 1
+elif [ $HTTP_RESPONCE_CODE_EN -eq 404 ]; then
+  echo "$RELEASE_BLOG_URL_EN is not found. This URL is incorrect."
+  exit 1
+fi
+
+ANNOUNCE_JA_TEMPLATE="$PGROONGA_LATEST_VERSION_AND_DATE リリース！ $RELEASE_BLOG_URL_JA 今回のリリースでは、"
+ANNOUNCE_EN_TEMPLATE="$PGROONGA_LATEST_VERSION_AND_DATE has been released! $RELEASE_BLOG_URL_EN In this release,"
+
+echo $ANNOUNCE_JA_TEMPLATE; echo $ANNOUNCE_EN_TEMPLATE
```
Comment (kou) [inline tools/create-release-announce-template-for-twitter.sh:29]:
```suggestion
echo "$PGROONGA_LATEST_VERSION_AND_DATE リリース！ $RELEASE_BLOG_URL_JA 今回のリリースでは、"
echo "$PGROONGA_LATEST_VERSION_AND_DATE has been released! $RELEASE_BLOG_URL_EN In this release,"
```

Comment (komainu8) [reply tools/create-release-announce-template-for-twitter.sh]: Thanks.
I've fixed it.

**Pull request merged**